# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
float64 left_finger
float64 right_finger
float64 left_claw
float64 right_claw
